src/sentry/attachments/redis.py

```diff
@@ -15,12 +15,12 @@ def __init__(self, **options):
         cluster_id = options.pop("cluster_id", None)
         if cluster_id is None:
             cluster_id = getattr(settings, "SENTRY_ATTACHMENTS_REDIS_CLUSTER", "rc-short")
-        BaseAttachmentCache.__init__(self, inner=RedisClusterCache(cluster_id, **options))
-
-
-class RbAttachmentCache(BaseAttachmentCache):
-    def __init__(self, **options):
-        BaseAttachmentCache.__init__(self, inner=RbCache(**options))
+        BaseAttachmentCache.__init__(
+            self,
+            inner=RedisClusterCache(
+                cluster_id, version=settings.ATTACHMENT_CACHE_VERSION, **options
```

Comment:
I would be tempted to put a comment right above this saying 'See the `ATTACHMENT_CACHE_VERSION` to understand why we do not use the global `CACHE_VERSION` here`.

Reply:
Since I moved the constant to the same file I think we probably don't need this comment now?

Reply:
Ah OK you're right

Reply:
Dunno why I totally overlooked that haha